src/Apps/MyCollection/Routes/EditArtwork/Mutations/useDeleteArtworkImage.ts

```diff
@@ -0,0 +1,24 @@
+import { graphql } from "relay-runtime"
+import { useMutation } from "Utils/Hooks/useMutation"
+import { useDeleteArtworkImageMutation } from "__generated__/useDeleteArtworkImageMutation.graphql"
+
+export const useDeleteArtworkImage = () => {
+  return useMutation<useDeleteArtworkImageMutation>({
+    mutation: graphql`
+      mutation useDeleteArtworkImageMutation($input: DeleteArtworkImageInput!) {
+        deleteArtworkImage(input: $input) {
+          artworkOrError {
+            ... on ArtworkMutationDeleteSuccess {
+              success
```

Comment:
I've tried to return it, but it doesn't seem to have an effect.

Reply:
I was thinking we need to do that to avoid having the logic we have in the app to refresh the artwork page manually. I will add a ticket for it to do it in MP

Reply:
here is it https://artsyproduct.atlassian.net/browse/CX-2829